### PR TITLE
Cleanup documentation

### PR DESCRIPTION
**Pull request by arcalinea**
Added zcash developer public keys to /contrib/gitian-downloader, incorporated some of @daira's suggestions for files in the /doc directory

### CODE REVIEW

File: doc/dnsseed-policy.md [hunk 1]
```diff
@@ -44,13 +42,11 @@ details of their operating practices.
 related to the DNS seed operation.
 
 If these expectations cannot be satisfied the operator should
-discontinue providing services and contact the active Bitcoin
-Core development team as well as posting on
-[bitcoin-dev](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev).
+discontinue providing services and contact the active Zcash development team as well as creating an issue in the [Zcash repository](https://github.com/Zcash/zcash).
```
Comment (daira) [inline doc/dnsseed-policy.md]: Lowercase zcash in the URL; also please wrap the line.

**Comment by daira** [comment]
ACK. Did not check the PGP keys.

File: doc/dnsseed-policy.md (hunk 1 above)
Comment (bitcartel) [inline doc/dnsseed-policy.md]: @arcalinea Once this comment is addressed, I will ACK and merge - if this is for RC1 (I don't see a milestone or anything on the ticket).

Comment (daira) [inline doc/dnsseed-policy.md]: This can be merged for rc1, although it doesn't block the release.

**Comment by ageis** [comment]
Ideally people should be using the same keys that are in `gitian-downloader` to sign the assertions/manifests that are pushed to https://github.com/zcash/gitian.sigs. I'm not sure how this directory integrates with any tooling though, it presently doesn't, as far as I can see.

**Comment by str4d** [comment]
@arcalinea please address https://github.com/zcash/zcash/pull/1532#discussion_r83387896 :)

@ageis I'm not sure what the integration is either. Are the key names here supposed to match the directories used in gitian.sigs? Either way, that does not block this PR; we can fix integration in a later PR.

**Comment by ageis** [comment]
@str4d It's probably so that people have quick access to the keys so that they can run `gverify`. Requires manually importing them right now though. I have a branch in progress that will include all our keys in the Vagrant user's keyring. 

```vagrant@zcash-build:~/gitian-builder$ ./bin/gverify -v -d ../gitian.sigs/ -r v1.0.0-rc1 ../zcash/contrib/gitian-descriptors/gitian-linux.yml
gpg: Signature made Mon 17 Oct 2016 09:50:30 PM GMT
gpg:                using RSA key F87592F35333D44F
gpg: Good signature from "Kevin M. Gallagher <kevingallagher@gmail.com>" [unknown]
gpg:                 aka "Kevin M. Gallagher <kevin@z.cash>" [unknown]
gpg:                 aka "Kevin M. Gallagher <kevinmg@riseup.net>" [unknown]
gpg:                 aka "Kevin M. Gallagher <kevin@transparencytoolkit.org>" [unknown]
gpg:                 aka "Kevin M. Gallagher <kevin@libraryfreedomproject.org>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 2C84 664F 26AA E27B AD57  90FD B604 C32A D5D7 C6D8
     Subkey fingerprint: A3F2 B50A 4386 2FE2 E5E6  F270 F875 92F3 5333 D44F
kevin: OK

gpg: Signature made Tue 18 Oct 2016 02:52:45 AM GMT
gpg:                using RSA key 6A6914DAFBEA00DA
gpg: Good signature from "str4d (http://str4d.i2p) <str4d@mail.i2p>" [unknown]
gpg:                 aka "Jack Grigg <jack@z.cash>" [unknown]
gpg:                 aka "str4d <str4d@i2pmail.org>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 2253 E2A1 EEB4 0E2A 3D22  EB1D 0EC5 1FCD A94F B53E
     Subkey fingerprint: 2293 0E17 090A 7ABD 9525  CA8E 6A69 14DA FBEA 00DA
jack@z.cash: OK
```

**Comment by daira** [comment]
self-ACK febe591 (checked rendered version). @zkbot r+

**Pull request merged**